content/react/concepts/routing/routing.md

```diff
@@ -29,19 +29,33 @@ npm install react-router-dom
 
```

Comment:
```suggestion

The newest version of React Router is v6, which comes with many breaking changes including: 

- Using a `<Routes />` component instead of a `<Switch />` component for `<Route />` configuration.
- Using the `element` attribute instead of `component` when passing a JSX to a `<Link />`.
```